### PR TITLE
[BUG-3791] Adds missing permissions in logstash static role

### PR DESCRIPTION
**Pull request by dpavlos**
### Description
* Category: Bug fix

### Issues Resolved
- https://github.com/opensearch-project/security/issues/3791

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: src/main/resources/static_config/static_roles.yml [hunk 1]
```diff
@@ -132,6 +132,7 @@ logstash:
   index_permissions:
     - index_patterns:
         - "logstash-*"
+        - "ecs-logstash-*"
```
Comment (derek-ho) [inline src/main/resources/static_config/static_roles.yml:135]: I am wondering whether ecs-logstash-* is a industry-standard index that is created/if it isn't then maybe this should be added via another role?

Comment (derek-ho) [reply src/main/resources/static_config/static_roles.yml:135]: Just re-read the issue and I don't see any harm in adding this, if it is supported by logstash-output-opensearch plugin

**Review by derek-ho — APPROVED** [review]

**Review by cwperks — APPROVED** [review]

File: src/main/resources/static_config/static_roles.yml (hunk 1 above)
Comment (dpavlos) [reply src/main/resources/static_config/static_roles.yml:135]: Thanks @derek-ho and @cwperks for reviewing this. Is it OK to resolve this thread?

Comment (derek-ho) [reply src/main/resources/static_config/static_roles.yml:135]: Yes please, go ahead. I can't resolve it from my end unfortunately.

**Review by DarshitChanpura — APPROVED** [review]

**Pull request merged**